test/tests/disclosure-navigation.js

```diff
@@ -0,0 +1,264 @@
+'use strict';
+
+const { ariaTest } = require('..');
+const { By, Key } = require('selenium-webdriver');
+const assertAriaControls = require('../util/assertAriaControls');
+const assertAttributeValues = require('../util/assertAttributeValues');
+const assertTabOrder = require('../util/assertTabOrder');
+const assertHasFocus = require('../util/assertHasFocus');
+
+const exampleFile = 'disclosure/disclosure-navigation.html';
+
+const ex = {
+  buttonSelector: '#exTest button',
+  menuSelector: '#exTest > li > ul',
+  linkSelector: '#exTest > li > ul a',
+  buttonSelectors: [
+    '#exTest > li:nth-child(1) button',
+    '#exTest > li:nth-child(2) button',
+    '#exTest > li:nth-child(3) button'
+  ],
+  menuSelectors: [
+    '#exTest > li:nth-child(1) ul',
+    '#exTest > li:nth-child(2) ul',
+    '#exTest > li:nth-child(3) ul'
+  ]
+};
+
+// Attributes
+
+ariaTest('"aria-controls" attribute on button', exampleFile, 'button-aria-controls', async (t) => {
+  t.plan(1);
+  await assertAriaControls(t, ex.buttonSelector);
+});
+
+ariaTest('"aria-expanded" attribute on button', exampleFile, 'button-aria-expanded', async (t) => {
+  t.plan(7);
+
+  await assertAttributeValues(t, ex.buttonSelector, 'aria-expanded', 'false');
+
+  let buttons = await t.context.session.findElements(By.css(ex.buttonSelector));
+  let menus = await t.context.session.findElements(By.css(ex.menuSelector));
+  for (let i = buttons.length - 1; i >= 0; i--) {
+    await buttons[i].click();
+    t.true(
+      await menus[i].isDisplayed(),
+      'Each dropdown menu should display after clicking its trigger'
+    );
+    await assertAttributeValues(t, ex.buttonSelectors[i], 'aria-expanded', 'true');
+  }
+});
+
+ariaTest('"aria-current" attribute on links', exampleFile, 'link-aria-current', async (t) => {
+  t.plan(36);
+
+  const buttons = await t.context.session.findElements(By.css(ex.buttonSelector));
+  const menus = await t.context.session.findElements(By.css(ex.menuSelector));
+
+  for (let b = 0; b < buttons.length; b++) {
+    const links = await menus[b].findElements(By.css('a'));
+
+    for (let l = 0; l < links.length; l++) {

```

Comment:
I don't want to block merging, just have an optional suggestion: the javascript that controls `aria-current` will function the same regardless of which link is clicked, so it's only necessary to test on one link click and cut down on test time a little.

Otherwise, looks good! Thanks for writing this! (Also, changing the loops is entirely up to you, I don't want to hold up merging the PR)